src/loginComponents/checkRoles.js

```diff
@@ -11,10 +11,7 @@ export class AuthorizedElement extends React.Component {
   }
 
   render() {
-    if (
-      this.props.fatherState.keycloak.hasRealmRole(this.props.roleNeeded) &&
-      this.props.fatherState.groups.indexOf(this.props.groupNeeded) > -1
-    ) {
+    if (typeof this.props.tenantNeeded === 'undefined' || this.props.tenantNeeded === null || this.props.iSuperAdmin) {
```

Comment:
the idea is that you do the check here if based on the tokenDecoded the user is super admin or not.